### DISCUSSION TITLE
Convert AWAY from Docker...

### DISCUSSION THREAD

**Discussion by sntlewis** [Donkie/Spoolman#213] [Q&A] [answered]
IF (and it is a big if) I am understanding the latest changelog (please correct me if I'm wrong), Spoolman can run now without Docker.

If so, what do I need to do to move and install Spoolman outside of docker?  Is it worth my effort when I can simply run the update commands.

**Comment by Donkie** [comment, marked as answer]
If you're already using docker then I recommend you keep on using it, since it's easier to keep it contained and updated.

If you really want to move to standalone, I can give you some tips

**Comment by sntlewis** [comment]
I'm good staying with Docker since it is setup and running well.
